mail_force_email_notification/tests/test_mail_thread.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2025 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import tagged
+from odoo.tests.common import TransactionCase
+
+
+@tagged("-at_install", "post_install")
+class TestMailThread(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

Comment:
Could leverage `BaseCommon` class to remove this line

Reply:
not a blocker